models/Achievement.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Achievement Model
+ *
+ * Optional feature.
+ * Represents a possible achievement users can earn.
+ */
+
+module.exports = (sequelize, Sequelize) => {
+  const { DataTypes } = Sequelize;
+
+  const Achievement = sequelize.define(
+    "Achievement",
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      requirements: {
+        type: DataTypes.TEXT,
+      },
+      iconUrl: {
+        type: DataTypes.STRING,
+      },
```

Comment:
Perhaps an image url rather than an icon
```suggestion
    imageUrl: {
      type: DataTypes.STRING,
    },
```
we can store a range of images in the public folder for the front end to reference, make a fun little design task for someone.